client/packages/invoices/src/Prescriptions/api/hooks/usePrescription.ts

```diff
@@ -135,7 +135,8 @@ const useGetById = (invoiceId: string | undefined) => {
     if (invoice?.__typename === 'InvoiceNode') {
       return invoice;
     } else {
-      throw new Error('Could not find invoice');
+      console.log('No invoice found', invoiceId);
```

Comment:
Though, this fails compilation step. I think you can use `console.error()` instead?